guides/prefix-cache-storage/cpu/README.md

```diff
@@ -96,8 +96,79 @@ NAME                                  READY   STATUS    RESTARTS   AGE
 llm-d-infpool-epp-xxxxxxxx-xxxxx     1/1     Running   0          16m
 llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
 llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
+llm-d-model-server-xxxxxxxx-xxxxx   1/1     Running   0          11m
 ```
 
+## Benchmark
```

Comment:
Suggest moving this to the end of the doc, and perhaps with `##Appendix: Benchmark`